onnxruntime/core/providers/cpu/nn/layer_norm_impl.cc

```diff
@@ -11,6 +11,7 @@
 #include "core/providers/common.h"
 #include "core/util/force_inline.h"
 #include "core/util/math_cpuonly.h"
+#include <Eigen/Core>
```

Comment:
`<Eigen/Core>` is already included in `core/util/math_cpuonly.h`. there, it also has some handling for warnings. I'd suggest just using that so we don't need to duplicate the warning suppressions here too.